runners/google-cloud-dataflow-java/examples/build.gradle

```diff
@@ -42,57 +42,60 @@ def dockerImageName = project(':beam-runners-google-cloud-dataflow-java').ext.do
 // If -PuseExecutableStage is set, the use_executable_stage_bundle_execution wil be enabled.
 def fnapiExperiments = project.hasProperty('useExecutableStage') ? 'beam_fn_api,use_executable_stage_bundle_execution' : "beam_fn_api"
 
+// we require dataflowWorker, optional workerHarnessContainerImage and optional additionalOptions
+def commonConfig = { dataflowWorkerJar, workerHarnessContainerImage = '', additionalOptions = [] ->
+   // return the preevaluated configuration closure
+           return {
```

Comment:
will this ident pass spot bugs?

Reply:
Good catch. It did but IMO it shouldn't have. It looks like no build.gradle files are checked only when applyGroovyNature() is invoked in them. Looks like a bug.

Reply:
https://issues.apache.org/jira/browse/BEAM-7040 issue for this

@mwalenia could you fix it according to Intelij's suggestions?